ext/SparseConnectivityTracerSpecialFunctionsExt/SparseConnectivityTracerSpecialFunctionsExt.jl

```diff
@@ -0,0 +1,115 @@
+module SparseConnectivityTracerSpecialFunctionsExt
+
+if isdefined(Base, :get_extension)
+    import SparseConnectivityTracer as SCT
+    using SpecialFunctions
+else
+    import ..SparseConnectivityTracer as SCT
+    using ..SpecialFunctions
+end
+
+#=
+Complex functions are ignored.
+Functions with more than 2 arguments are ignored.
+Functions with integer arguments are ignored.
```

Comment:
Why ignore functions with integer arguments?
While they are non-differentiable, they are still useful for `ConnectivityTracer`.

Reply:
Because integer arguments require special cases, like the `^` shenanigans we had to deal with in #83. And because I'm not well-versed in SpecialFunctions. So I'd rather have something error than be implemented incorrectly. We can always improve later

Reply:
I agree with the second point, but `^` is related to the exponent having a multitude of different methods depending on argument types, leading to ambiguities if you only specify the concrete type of one of the two arguments.